test/e2e-common.sh

```diff
@@ -87,6 +87,13 @@ function knative_setup() {
   unleash_duck || fail_test "Could not unleash the chaos duck"
 
   install_cert_manager || fail_test "Could not install Cert Manager"
+
+  install_prometheus || fail_test "Could not install Prometheus"
+}
+
+function install_prometheus() {
+    helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+    helm install prometheus prometheus-community/prometheus
```

Comment:
Could you install prometheus directly without helm? Perhaps from [here](https://github.com/prometheus-operator/kube-prometheus)? As far as I know the test environment does not have helm installed